rafthttp/peer.go

```diff
@@ -271,12 +264,14 @@ func (p *peer) pick(m raftpb.Message) (writec chan<- raftpb.Message, picked stri
 	// stream for a long time, only use one of the N pipelines to send MsgSnap.
 	if isMsgSnap(m) {
 		return p.pipeline.msgc, pipelineMsg
-	} else if writec, ok = p.msgAppWriter.writec(); ok && canUseMsgAppStream(m) {
```

Comment:
`msgAppWriter` is used for msgAppV2 stream after change. It deserves a better name `msgAppV2Writer` to make it clear.

Originally `msgAppWriter` is used for both msgApp and msgAppV2 stream.